sdk-dotnet/src/LittleHorseSDK/LittleHorse.Worker/Internal/Helpers/LHMappingHelper.cs

```diff
@@ -0,0 +1,168 @@
+        }
+        public static string ObjectSerializeToJson(object o)
+        {
+            var jsonSettings = new JsonSerializerSettings();
+
+            return JsonConvert.SerializeObject(o, jsonSettings);
+        }
+        public static object? DeserializeFromJson(string json, System.Type type)
+        {
+            return JsonConvert.DeserializeObject(json, type);
+        }
+        public static VariableValue MapObjectToVariableValue(object? o)
+        {
+            if (o is VariableValue variableValue) return variableValue;
+
+            var result = new VariableValue();
+            if (o == null)
+            {
+                result.Type = VariableType.Null;
+            }
+            else if (o is long longValue)
+            {
+                result.Type = VariableType.Int;
+                result.Int = longValue;
+            }
+            else if (o is int intValue)
+            {
+                result.Type = VariableType.Int;
+                result.Int = intValue;
+            }
+            else if (o is double doubleValue)
+            {
+                result.Type = VariableType.Double;
+                result.Double = doubleValue;
+            }
+            else if (o is float floatValue)
+            {
+                result.Type = VariableType.Double;
+                result.Double = floatValue;
+            }
+            else if (o is string stringValue)
+            {
+                result.Type = VariableType.Str;
+                result.Str = stringValue;
+            }
+            else if (o is bool boolValue)
+            {
+                result.Type = VariableType.Bool;
+                result.Bool = boolValue;
+            }
+            else if (o is byte[] byteArray)
+            {
+                result.Type = VariableType.Bytes;
+                result.Bytes = ByteString.CopyFrom(byteArray);
+            }
+            else
+            {
+                // At this point, all we can do is try to make it a JSON type.
+                var jsonStr = ObjectSerializeToJson(o);
+
+                if (o is IList<object> list)
+                {
+                    result.Type = VariableType.JsonArr;
+                    result.JsonArr = jsonStr;
+                }
+                else
+                {
+                    result.Type = VariableType.JsonObj;
+                    result.JsonObj = jsonStr;
+                }
+            }
+
+            return result;
+        }
+        public static VariableValue MapExceptionToVariableValue(Exception exception, LHWorkerContext ctx) 
+        {
+            using (StringWriter sw = new StringWriter())
+            {
+                using (TextWriter pw = sw)
+                {
+                    pw.WriteLine(exception.StackTrace);
+                }
+
+                string output = sw.ToString();
+
+                if (ctx.LogOutput != null)
+                {
+                    output += "\n\n\n\n" + ctx.LogOutput;
```

Comment:
@eduwercamacaro Looks like when you're back from vacation you get to update this here along with Go :smile: